security/advancedtls/testdata/crl/README.md

```diff
@@ -116,4 +119,14 @@ cert are below.
 $ openssl ca -gencrl -keyfile provider_client_trust_key.pem -cert provider_client_trust_cert.pem -out provider_crl_empty.pem -config provider_crl.cnf
 $ openssl ca -revoke provider_server_cert.pem -keyfile provider_client_trust_key.pem -cert provider_client_trust_cert.pem -config provider_crl.cnf
 $ openssl ca -gencrl -keyfile provider_client_trust_key.pem -cert provider_client_trust_cert.pem -out provider_crl_server_revoked.pem -config provider_crl.cnf
-```
\ No newline at end of file
+```
+
+The commands to generate CRL file by 'malicious' CA are below.
+```
+$ openssl genrsa -out provider_malicious_client_trust_key.pem 4096
```

Comment:
I think it's best to have this entirely in a script and just link the script, rather than having terminal commands in the README

Reply:
It looks like you do have it in a script already, so by having it here we're just creating a future documentation problem for ourselves where we might not update both things at the same time, let's have one authoritative source of truth for the creation, and that should be a script. WDYT?

Reply:
I think atm it's intermediate state. We want to have a script just to 'do all the setup' but it's tough to understand the intent. That's why I created a README with some explanations there.
I think the end goal would be to drop the README in favor of actual documentation once it's ready, so I'd prefer to keep the README for now

Reply:
But right now it's an exact duplicate of the script, right? That's a maintenance headache - there's no reason to copy that to the README. I think we should only have bash code in the README that aren't covered in the scripts

Reply:
refactored